bootstrap.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+apt-get update
+apt-get install python-software-properties -y
+apt-add-repository ppa:brightbox/ruby-ng -y
+apt-get update
+apt-get install ruby2.3 ruby2.3-dev mongodb-server make -y
+gem install bundler
```

Comment:
You might want to use our **Ansible** script for **Ruby** from the **Ministers Under the Influence Project** https://github.com/HackBrexit/MinistersUnderTheInfluence/pull/51/files#diff-22890c427500c30c9f0f6b951a0f60c9